scripts/amd/setup_rocm_libs.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CWD=$(dirname "$0")
+
+echo "Collecting Necessary ROCM libraries"
+if [[ -z "${ROCM_VERSION}" ]]; then
+  ROCM_VERSION=5.4.2
+fi
+
+if [[ -z "${TRITON_ROCM_DIR}" ]]; then
+  TRITON_ROCM_DIR=python/triton/third_party/rocm
+fi
+  
```

Comment:
Potentially add flag to allow gathering rocm libraries from local directories in case we have versions that aren't in repo.radeon.com just yet